.github/renovate.json

```diff
@@ -7,7 +7,7 @@
         "dependencies"
     ],
     "npm": {
-        "stabilityDays": 1
+        "minimumReleaseAge": "3 days"
     },
```

Comment:
Please remove all these lines, the syntax has changed so these are no longer relevant

Reply:
Referring to L9-11